homeassistant/helpers/config_validation.py

```diff
@@ -402,3 +402,9 @@ def validator(value):
     [vol.Any(SERVICE_SCHEMA, _SCRIPT_DELAY_SCHEMA, EVENT_SCHEMA,
              CONDITION_SCHEMA)],
 )
+
+
+def string_schema(*args: Sequence) -> Dict:
+    """Return a schema dict of Required values or Optional values."""
+    return {val if isinstance(val, vol.Optional) else vol.Required(val): string
+            for val in args}
```

Comment:
@Teagan42, opted for this method instead of creating a USER_PASS_HOST_SCHEMA (it was too specific)

It allows quick creation of all string schemas, i.e 
```python
PLATFORM_SCHEMA.extend(string_schema(CONF_USERNAME, CONF_PASSWORD, CONF_HOST))
```
instead of
```python
PLATFORM_SCHEMA.extend(USER_PASS_HOST_SCHEMA.schema)
```

Comments welcome